backend/core/src/units/units.dto.ts

```diff
@@ -0,0 +1,88 @@
+import { Expose } from "class-transformer"
+import { IsDateString, IsNumber, IsString, IsUUID } from "class-validator"
+
+export class UnitDto {
+  @Expose()
+  @IsString()
+  @IsUUID()
+  id: string
+
+  @Expose()
+  @IsString()
+  amiPercentage: string
+
+  @Expose()
+  @IsString()
+  annualIncomeMin: string
```

Comment:
This might not be the right time to fix this, but the data types of this model have always seemed off to me - it seems like these should be numeric?